Networking/SR Linux/ci/Jenkinsfile

```
@@ -65,7 +65,7 @@ pipeline {
                         sh '''
                             # Cleanup before starting
                             cd "Networking/SR Linux"
-                            sudo clab destroy --cleanup -t containerlab/topology.yml
+                            sudo clab destroy --cleanup -t containerlab/topology.yml | true
```

Minor: Why use a pipe (`|`) an not a or (`||`)?  It does the same here, but feels strange.  Is the idea to hide any error messages we don't care about?

---

Damn, meant to use `||`, thanks.